### PR TITLE
udp num dst

### PR DESCRIPTION
**Pull request by thattil**
->udp_numdst readonly
->clearudpdst to clear all udp destinations

### CODE REVIEW

File: slsDetectorServers/jungfrauDetectorServer/slsDetectorServer_defs.h [hunk 1]
```diff
@@ -67,9 +67,9 @@ enum DACINDEX {
 #define NSPECIALDACS                         (3)
 #define SPECIALDACINDEX                      {J_VREF_PRECH, J_VREF_DS, J_VREF_COMP};
 #define SPECIAL_DEFAULT_DYNAMIC_GAIN_VALS                                      \
-    { 1000, 500, 400 }
+    { 1450, 480, 420 }
```
Comment (erikfrojdh) [inline slsDetectorServers/jungfrauDetectorServer/slsDetectorServer_defs.h:70]: I didn't expect this to change in this PR. Which settings are new and which are old?

File: slsDetectorSoftware/include/sls/Detector.h [hunk 2]
```diff
@@ -671,8 +671,7 @@ class Detector {
     /** [Jungfrau][Eiger] */
     Result<int> getNumberofUDPDestinations(Positions pos = {}) const;
 
-    /**[Jungfrau][Eiger] Options 1-32 */
-    void setNumberofUDPDestinations(const int value, Positions pos = {});
+    void clearUDPDestination(Positions pos = {});
```
Comment (erikfrojdh) [inline slsDetectorSoftware/include/sls/Detector.h:674]: Maybe clearUDPDestinations?

**Review by erikfrojdh — COMMENTED** [review]
Two minor comments otherwise good!

File: slsDetectorServers/jungfrauDetectorServer/slsDetectorServer_defs.h (hunk 1 above)
Comment (thattil) [reply slsDetectorServers/jungfrauDetectorServer/slsDetectorServer_defs.h:70]: Yes, this and G2 are the things I am adding in here.

File: slsDetectorSoftware/include/sls/Detector.h (hunk 2 above)
Comment (thattil) [reply slsDetectorSoftware/include/sls/Detector.h]: done

**Pull request merged**